packages/transport/src/api/udp.ts

```diff
@@ -37,14 +41,7 @@ export class UdpApi extends AbstractApi {
     public write(path: string, buffer: Buffer, signal?: AbortSignal) {
         const [hostname, port] = path.split(':');
 
-        return new Promise<
-            ResultWithTypedError<
-                undefined,
-                | typeof ERRORS.INTERFACE_DATA_TRANSFER
-                | typeof ERRORS.UNEXPECTED_ERROR
-                | typeof ERRORS.ABORTED_BY_SIGNAL
-            >
-        >(resolve => {
```

Comment:
not 100% sure by this change. in fact I am lying a bit here, this very promise can't return all of the errors specified by the type. but it is probably ok, this type is intended to be consumed by a higher layer so we probably don't need to be extremely specific here.

Reply:
lgtm